crdt/atom/atom_id.go

```diff
@@ -0,0 +1,90 @@
+package atom
+
+import (
+	"fmt"
+
+	"github.com/crdteam/causal-tree/crdt/utils/indexmap"
+)
+
+// AtomID is the unique identifier of an atom.
+type AtomID struct {
```

Comment:
This can be renamed to `ID`, to be referenced as `atom.ID`

Reply:
done.